bundles/org.openhab.binding.clementineremote/README.md

```diff
@@ -0,0 +1,334 @@
+# ClementineRemote Binding
+
+<img style="float: right; margin: 0 0 5px 5px" src="doc/screenshot.png" alt="Screenshot" />
+This binding shall bring the benefits of over-the-network control of the [Clementine-Player] to OpenHab!
+
+It allows control of playback, position and volume and allows display of various title information:
+
+* Artist
+* Album
+* Title
+* current position within track
+* Album cover
+
+
+[→ useful Links][links]
```

Comment:
I'm not aware of other bindings supplying this kind of links. 
If it has to stay here, maybe change it to Developer links as only developer related links are provided. But maybe it is better to embed the link to the Clementine Dev/Api docs in the description at the top with some text to show that it is based in the official API. (just a thought)

Reply:
Good suggestion. I removed the file and added a link to the JavaDoc comment of the ClementineRemoteHandler class.